### ISSUE TITLE
broken link in codetour

### ISSUE THREAD

**Issue by josef-widder** [informalsystems/quint#556]
I am doing the codetour for "hello, world", and on page #1 the link to `hello.qnt` seems broken.

**Comment by thpani** [comment]
All relative markdown links are broken in CodeTour, e.g., also the images:

<img width="260" alt="Screen Shot 2023-02-03 at 14 21 43" src="https://user-images.githubusercontent.com/82047/216614489-5caa3117-c40f-4f68-adc7-7683b4d9aff0.png">

**Comment by thpani** [comment]
@konnov I think #695 didn't actually update the tour. Run `make`? 🙃

**Comment by konnov** [comment]
This seems to be fixed long time ago

**Comment by thpani** [comment]
> This seems to be fixed long time ago

Not true, both the link reported by Josef and [the image](https://github.com/informalsystems/quint/issues/556#issuecomment-1415868525) are broken (opening VSCode inside `./tutorials`).

**Comment by konnov** [comment]
This is a zombie issue, resurrecting two times :)
